optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeUserTaskImportIT.java

```diff
@@ -917,6 +950,11 @@ private OffsetDateTime getTimestampForAssignedUserTaskEvents(
     return getTimestampForFirstZeebeEventsWithIntent(eventsForElement, ASSIGNED);
   }
 
+  private OffsetDateTime getTimestampForAssignedLastUserTaskEvents(
+      final List<ZeebeUserTaskRecordDto> eventsForElement) {
+    return getTimestampForLastZeebeEventsWithIntent(eventsForElement, ASSIGNED);
+  }
```

Comment:
NABD but not sure this helper adds much value, as the proxied method already has reasonably good naming and interface